be/src/cloud/config.cpp

```diff
@@ -75,4 +75,6 @@ DEFINE_mInt32(tablet_txn_info_min_expired_seconds, "120");
 
 DEFINE_mBool(enable_use_cloud_unique_id_from_fe, "true");
 
+DEFINE_mBool(enable_cloud_delete_old_version_delete_bitmap, "false");
```

Comment:
`enable_cleanup_delete_bitmap_on_compaction`
don't need to specify `cloud` in the config name, since we also support this feature on non-cloud env(another PR)